submissions/5mountains/popup/index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>pure html/css Popup</title>
+    <link rel="icon" href="./img/favicon.png" type="image/png">
+    <link rel="stylesheet" href="./css/main.css">
+</head>
+<body>
+    <nav class="nav">
+        <!-- main menu -->
+        <ul class="nav__menu">
```

Comment:
Don't forget about comments

Reply:
I thought that I already removed all comments! But let me double-check!

Reply:
It has also been done!